plugins/modules/host.py

```diff
@@ -225,6 +381,39 @@ def main():
             provision_method=dict(choices=['build', 'image', 'bootdisk']),
             image=dict(type='entity', scope=['compute_resource']),
             compute_attributes=dict(type='dict'),
+            interfaces_attributes=dict(type='list', elements='dict', options=dict(
+                mac=dict(),
+                ip=dict(),
+                ip6=dict(),
+                type=dict(default='interface', choices=['interface', 'bmc', 'bond', 'bridge']),
```

Comment:
if I read https://theforeman.org/api/2.1/apidoc/v2/hosts/create.html correctly, the `interface` default is present server-side and we've been usually not setting it *additionally* as a default for such cases in the modules.
```suggestion
                type=dict(choices=['interface', 'bmc', 'bond', 'bridge']),
```